src/astro/utils/sql_handler.py

```diff
@@ -0,0 +1,97 @@
+from typing import Dict, Optional
+
+from airflow.exceptions import AirflowException
+from airflow.models import BaseOperator
+from sqlalchemy.sql.functions import Function
+
+from astro.databases.base import BaseDatabase
+from astro.settings import SCHEMA
+from astro.sql.table import Metadata, Table
+
+
+class SQLHandler(BaseOperator):
```

Comment:
How much value is this class giving? Perhaps we could allow the operators to interact with the database directly and decide when/if to try to create a schema.